lib/usd/ui/layerEditor/mayaCommandHook.cpp

```diff
@@ -34,6 +36,9 @@
 #define STR(x) std::string(x)
 
 namespace {
+
+PXR_NAMESPACE_USING_DIRECTIVE;
```

Comment:
Why did you add this? According to our coding standards we should use the fully qualified name. So for USD code that would be "PXR_NS::xxx"

Reply:
Mostly because trhe UsdMayaUtil is under the PXR_NS, which is weird to me, so if someone fixes that then its one spot to change. But anyhow I put PXR_NS:: before my UsdMayaUtil calls so its not an issue. There are two of them.